docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/cluster_limitations.md

```diff
@@ -1,20 +1,20 @@
 
-    !!! note
-        A cluster can be created which contains only one HCL Portal server, enabling a single HCL Portal server to be operational in a managed cell.
+    !!! Note
+        A cluster can be created that contains only one HCL Portal server, enabling a single HCL Portal server to operate in a managed cell.
 
--   You cannot use the Derby database. The HCL Portal databases must be transferred to a supported external database, for example: DB2®, Oracle, or SQL Server.
--   In a clustered environment, it is not possible to change settings through the Global Settings portlet or the XML configuration interface. These changes must be made by modifying the respective properties in the WebSphere® Integrated Solutions Console.
+-   You cannot use the Derby database. The HCL Portal databases must be transferred to a supported external database, for example, DB2®, Oracle, or SQL Server.
+-   In a clustered environment, you cannot change settings through the Global Settings portlet or the XML configuration interface. You must change the configuration by modifying the respective properties in the WebSphere® Integrated Solutions Console.
 -   To support search, you must install and configure a remote search service on an application server node that is not part of the cluster.
--   Administrative actions for HCL Digital Experience are immediately visible for the user who completes them. However, another user can be assured of seeing the changes only if the user logs out of HCL Digital Experience and then logs back in. This limitation applies to both cluster and non-cluster environments.
+-   Administrative actions for HCL Digital Experience are immediately visible to the user who completes them. However, another user can see the changes only after the user logs out of HCL Digital Experience and then logs back in. This limitation applies to both cluster and non-cluster environments.
 -   When you create a cluster or a cluster member, do not use spaces in the cluster name or the cluster member name.
 -   For the deployment manager and each HCL Portal node to be in the cluster, verify that each system clock is set to within 5 minutes of the others or the addNode command fails.
 
-**Restriction:** The serverName is hardcoded to HCL Portal. The serverName cannot be changed in a stand-alone environment. If you do change it, the ConfigEngine scripts do not work. For a clustered environment, see [Virtual Member Manager integration](../user_registry_consideration/plan_vmm_int.md) for options on replacing the HCL Portal JVM.
+**Restriction:** The `serverName` value is hardcoded in HCL Portal. You cannot change the `serverName` value in a stand-alone environment. If you do change it, the `ConfigEngine` scripts do not work. For a clustered environment, see [Virtual Member Manager integration](../user_registry_consideration/plan_vmm_int.md) for options on replacing the HCL Portal JVM.
```

Comment:
Kindly change HCL portal to HCL DX

Reply:
Use "serverName" as adjective and follow the adjective with a noun so the user knows what kind of thing is being discussed. serverName parameter. serverName setting. Whatever the correct noun is.